integration_tests/pages/admin/userManagement/showPage.ts

```diff
@@ -17,4 +21,36 @@ export default class ShowPage extends Page {
   shouldShowUserDetails(user: User): void {
     this.shouldContainSummaryListItems(userSummaryListItems(user))
   }
+
+  selectRoles(roles: ReadonlyArray<UserRole>): void {
```

Comment:
I think it'd be good to deselect all the roles here, and only select the ones we specify, just to give us confidence that the only roles we're sending are the ones we specify.